lib/astarte_device_sdk/data.c

```diff
@@ -170,21 +168,21 @@ static bool check_if_bson_type_is_mapping_type(
  ***********************************************/
 
 // clang-format off
-#define MAKE_INDIVIDUAL_FROM_FUNC(NAME, ENUM, TYPE, PARAM)                                         \
-    astarte_individual_t astarte_individual_from_##NAME(TYPE PARAM)                                \
+#define MAKE_FUNCTION_DATA_FROM(NAME, ENUM, TYPE, PARAM)                                           \
```

Comment:
What do you think about `MAKE_ASTARTE_DATA_FROM`?
```suggestion
#define MAKE_ASTARTE_DATA_FROM(NAME, ENUM, TYPE, PARAM)                                           \
```

Reply:
I fear it would be misleading to assume the function creates `astarte_data_t` from a type while it creates a function.